homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -313,7 +297,7 @@ def alarm_disarm(self, code: str | None = None) -> None:
 
     def alarm_arm_home(self, code: str | None = None) -> None:
         """Send arm home command."""
-        if self._code_arm_required and not self._validate_code(
+        if self.code_arm_required and not self._validate_code(
             code, STATE_ALARM_ARMED_HOME
```

Comment:
Think these should use the internal attribute.

Reply:
No?

![image](https://user-images.githubusercontent.com/195327/176185763-41c388d2-55d6-4dd9-adf3-fa9fc0c8a820.png)

Ref: <https://developers.home-assistant.io/docs/core/entity#entity-class-or-instance-attributes>

Reply:
I had missed that rule. Seem odd to me, thay would make changes to entity model affecting internal class behaviour. But if that is the recommended approach, it is. So lets change it back then.